awx/ui_next/src/types.js

```diff
@@ -229,3 +229,23 @@ export const User = shape({
   ldap_dn: string,
   last_login: string,
 });
+
+export const Group = shape({
+  id: number.isRequired,
+  type: oneOf(['group']),
+  url: string,
+  related: shape({}),
+  summary_fields: shape({}),
+  created: string,
+  modified: string,
+  name: string.isRequired,
+  description: string,
+  inventory: number,
+  variables: string,
+  has_active_failures: bool,
+  total_hosts: number,
+  hosts_with_active_failures: number,
+  total_groups: number,
+  groups_with_active_failures: number,
+  has_inventory_sources: bool,
```

Comment:
Some of these fields are going to be removed from the API: https://github.com/ansible/awx/pull/5448/files#diff-20967e9fd2543b95f7825c2e7ef5ec0bR15-R46

...so let's drop the deleted group fields from this `Group` data definition